.github/workflows/notebooks-release.yaml

```
@@ -0,0 +1,208 @@
+  USER_HASH: ${{ github.event.inputs.user-hash }}
+  REPO_OWNER: opendatahub-io
+  REPO_NAME: notebooks
+  VERSION: ${{ github.event.inputs.buildconfigs_version}}
+
+jobs:
+  #1. Update the params-latest.env and commit-latest.env files with new SHAs
+  Update-manifests:
+    uses: opendatahub-io/notebooks/.github/workflows/notebooks-digest-updater.yaml@main
+    with:
+      branch: ${{ github.event.inputs.branch }}
+      user-hash: ${{ github.event.inputs.user_hash }}
```

actionlint (1.7.7)

property "user_hash" is not defined in object type {branch: string; buildconfigs_version: string; release-type: string; release_name: string; tag: string; update_buildconfigs: string; user-hash: string}

(expression)